components/index/_shared/DataView.vue

```diff
@@ -248,6 +289,36 @@ export default Vue.extend({
     &--Additional {
       margin-bottom: 10px;
     }
+
+    &--Minimized-Additional {
+      display: -webkit-box;
+      -webkit-box-orient: vertical;
+      -webkit-line-clamp: 4;
```

Comment:
```suggestion
      display: flex;
      flex-direction: column;
      line-clamp: 4;
```
こちらですが、`box` が古いプロパティというのと `-webkit` といったprefixは付けない方向でいきたいので（autoprefixで自動でつくので）、`flex` 系でいければいきたいです。

Reply:
そして、`line-clamp` がIEで効かないっぽいので...（まだIE対応するのかという議論はさておいて）ちょっとここは対応方法を考えますね。

Reply:
少し調べましたが、 `line-clamp` は将来的なことを考慮すると使わないほうがいいっぽいですね...。ちょっと代替案を考えさせてください。

Reply:
@kaizumaki わかりました。ありがとうございます！